collector/lib/CollectorConfig.cpp

```diff
@@ -77,6 +79,8 @@ PathEnvVar tls_ca_path("ROX_COLLECTOR_TLS_CA");
 PathEnvVar tls_client_cert_path("ROX_COLLECTOR_TLS_CLIENT_CERT");
 PathEnvVar tls_client_key_path("ROX_COLLECTOR_TLS_CLIENT_KEY");
 
+PathEnvVar runtime_configmap_path("RUNTIME_CONFIGMAP_PATH");
```

Comment:
Rename this to `ROX_COLLECTOR_CONFIGURATION` or `ROX_COLLECTOR_CONFIGURATION_PATH` and provide a default value directly.
```suggestion
PathEnvVar config_file("ROX_COLLECTOR_CONFIGURATION", "/etc/stackrox/collector.json");
```